app/workers/scheduler/notifications_cleanup_scheduler.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Scheduler::NotificationsCleanupScheduler
+  include Sidekiq::Worker
+  include LowPriorityScheduler
+
+  TYPES_TO_CLEAN_UP = Notification::TYPES
```

Comment:
This would mean *all* types of notifications would get cleaned up. We discussed that some types, e.g. follow requests,  should maybe be excempt from this. This would be the place to codify this.